### ISSUE TITLE
Fix build of CLI library Docker image

### ISSUE THREAD

**Issue by garagatyi** [eclipse-che/che#7054]
### Description
In che6 branch build of CLI library, image fails with the following output: 
<details>

```
> websocket@1.0.23 install /runtime/node_modules/websocket
> (node-gyp rebuild 2> builderror.log) || (exit 0)

che-typescript@1.0.0 /runtime
+-- reflect-metadata@0.1.10 
`-- websocket@1.0.23 
  +-- debug@2.6.9 
  | `-- ms@2.0.0 
  +-- nan@2.7.0 
  +-- typedarray-to-buffer@3.1.2 
  | `-- is-typedarray@1.0.0 
  `-- yaeti@0.0.4 

npm WARN che-typescript@1.0.0 No repository field.
../lib-typescript/src/api/wsmaster/project/project.ts(51,120): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/api/wsmaster/project/project.ts(55,24): error TS2339: Property 'getRef' does not exist on type 'ServerDto'.
../lib-typescript/src/api/wsmaster/project/project.ts(56,38): error TS2339: Property 'getProperties' does not exist on type 'ServerDto'.
../lib-typescript/src/api/wsmaster/workspace/workspace.ts(65,67): error TS2694: Namespace '"/lib-typescript/src/api/dto/che-dto".org.eclipse.che.api.workspace.shared.dto' has no exported member 'ExtendedMachineDto'.
../lib-typescript/src/api/wsmaster/workspace/workspace.ts(65,133): error TS2339: Property 'ExtendedMachineDtoImpl' does not exist on type 'typeof dto'.
../lib-typescript/src/api/wsmaster/workspace/workspace.ts(74,83): error TS2339: Property 'EnvironmentRecipeDtoImpl' does not exist on type 'typeof dto'.
../lib-typescript/src/api/wsmaster/workspace/workspace.ts(79,105): error TS2339: Property 'machine' does not exist on type 'typeof api'.
../lib-typescript/src/api/wsmaster/workspace/workspace.ts(200,66): error TS2339: Property 'getRel' does not exist on type 'string'.
../lib-typescript/src/api/wsmaster/workspace/workspace.ts(201,38): error TS2339: Property 'getHref' does not exist on type 'string'.
../lib-typescript/src/internal/action/impl/create-start-workspace-action.ts(63,48): error TS2339: Property 'getRel' does not exist on type 'string'.
../lib-typescript/src/internal/action/impl/create-start-workspace-action.ts(64,43): error TS2339: Property 'getHref' does not exist on type 'string'.
../lib-typescript/src/internal/action/impl/execute-command-action.ts(68,65): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/action/impl/get-ssh-action.ts(87,51): error TS2694: Namespace '"/lib-typescript/src/api/dto/che-dto".org.eclipse.che.api' has no exported member 'machine'.
../lib-typescript/src/internal/action/impl/get-ssh-action.ts(87,125): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/action/impl/workspace-ssh-action.ts(85,78): error TS2694: Namespace '"/lib-typescript/src/api/dto/che-dto".org.eclipse.che.api.workspace.shared.dto' has no exported member 'ExtendedMachineDto'.
../lib-typescript/src/internal/action/impl/workspace-ssh-action.ts(105,21): error TS2322: Type 'Map<string, MachineDto>' is not assignable to type 'MachineDto[]'.
  Property 'length' is missing in type 'Map<string, MachineDto>'.
../lib-typescript/src/internal/action/impl/workspace-ssh-action.ts(106,50): error TS2694: Namespace '"/lib-typescript/src/api/dto/che-dto".org.eclipse.che.api' has no exported member 'machine'.
../lib-typescript/src/internal/action/impl/workspace-ssh-action.ts(106,127): error TS2339: Property 'getRuntime' does not exist on type 'MachineDto'.
../lib-typescript/src/internal/action/impl/workspace-ssh-action.ts(138,29): error TS2339: Property 'getConfig' does not exist on type 'MachineDto'.
../lib-typescript/src/internal/dir/che-dir.ts(485,34): error TS2339: Property 'getRel' does not exist on type 'string'.
../lib-typescript/src/internal/dir/che-dir.ts(486,27): error TS2339: Property 'getHref' does not exist on type 'string'.
../lib-typescript/src/internal/dir/che-dir.ts(739,50): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/dir/che-dir.ts(782,56): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/dir/che-dir.ts(785,53): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/dir/che-dir.ts(839,54): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/dir/che-dir.ts(903,56): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
../lib-typescript/src/internal/dir/che-dir.ts(910,53): error TS2339: Property 'getDevMachine' does not exist on type 'RuntimeDto'.
The command '/bin/sh -c cd /lib-typescript/ && npm install && npm test     && cd /runtime && npm install && /lib-typescript/node_modules/.bin/tsc --project /lib-typescript/     && mv /lib-typescript/lib /che-lib && cd /lib-typescript/src && find . -name "*.properties" -exec install -D {} /che-lib/{} \;    && rm -rf /lib-typescript && mv /runtime/node_modules /che-lib && rm -rf /runtime' returned a non-zero code: 2
✔  16:54 ~/workspace/che/dockerfiles/lib [ che6 | ⚑ 8 ] $
```
</details>

### Reproduction Steps

**OS and version:**    

**Diagnostics:**

**Comment by riuvshin** [comment]
that issue blocks release https://github.com/eclipse/che/issues/6938 6.0.0-M1 becuase Im not able to generate image

**Comment by skabashnyuk** [comment]
@benoitf any idea how to fix it?

**Comment by riuvshin** [comment]
again this blocks releases of CHE6.

**Comment by benoitf** [comment]
@skabasniuk IMHO it's to update code with new DTOs but I'm on PTO all this week

**Comment by skabashnyuk** [comment]
```update code with new DTOs```. Can you elaborate a  bit more what does it mean?

**Comment by slemeur** [comment]
Closed by #7846
